src/common/transformations/include/transformations/common_optimizations/mark_precision_sensitive_subgraphs.hpp

```diff
@@ -17,12 +17,20 @@ class TRANSFORMATIONS_API MarkPrecisionSensitiveSubgraphs;
 
 /**
  * @ingroup ie_transformation_common_api
- * @brief MarkPrecisionSensitiveSubgraphs transformation marks the constants
+ * @brief MarkPrecisionSensitiveSubgraphs transformation goes up and marks nodes
  * inside the subgraph starting from precision-sensitive input and ending at
  * the ShapeOf node as disabled for FP16 compression.
+ *
+ * If mark_only_consts is true only Consts are marked. False value is used
+ * during offline_transformations. For the GPU plugin need to keep
```

Comment:
Here you are saying that offline transformation uses `false` value but in the code I see the different thing: https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/pyopenvino/core/offline_transformations.cpp#L111

Reply:
Here is a typo in the comment. Code in `offline_tranformation` is correct one.
Thanks for noticing!